src/util/useDidUpdateEffect.ts

```diff
@@ -1,9 +1,5 @@
 /* eslint-disable header/header */
```

Comment:
This is missing a header

Reply:
Actually Thibaut disabled the header on purpose since that was taken from stackoverflow ( he copied the link ). Do you think that i should put the header anyway ?